.github/actions/repo-gardening/src/tasks/check-description/index.js

```diff
@@ -0,0 +1,266 @@
+/**
+ * External dependencies
+ */
+const moment = require( 'moment' );
+
+/**
+ * Internal dependencies
+ */
+const debug = require( '../../debug' );
+const getLabels = require( '../../get-labels' );
+const getNextValidMilestone = require( '../../get-next-valid-milestone' );
+
+/* global GitHub, WebhookPayloadPullRequest */
+
+/**
+ * Check if a PR has unverified commits.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Promise<boolean>} Promise resolving to boolean.
+ */
+async function hasUnverifiedCommit( octokit, owner, repo, number ) {
+	let isUnverified = false;
+
+	for await ( const response of octokit.paginate.iterator( octokit.pulls.listCommits, {
+		owner: owner.login,
+		repo,
+		pull_number: +number,
+	} ) ) {
+		response.data.map( commit => {
+			if ( commit.commit.message.includes( '[not verified]' ) ) {
+				isUnverified = true;
+			}
+		} );
+	}
+
+	return isUnverified;
+}
+
+/**
+ * Check for status labels on a PR.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Promise<boolean>} Promise resolving to boolean.
+ */
+async function hasStatusLabels( octokit, owner, repo, number ) {
+	const labels = await getLabels( octokit, owner, repo, number );
+	let hasStatus = false;
+
+	// We're really only interested in status labels
+	labels.map( label => {
+		if ( label.includes( '[Status]' ) ) {
+			debug( `check-description: this label (${ label }) includes the word Status` );
+			hasStatus = true;
+		}
+	} );
```

Comment:
We probably can replace `.map` with `.find`. It should simplify the method and will allow us to get rid of `hasStatus` variable

```suggestion
	labels.find( label => {
		if ( label.includes( '[Status]' ) ) {
			debug( `check-description: this label (${ label }) includes the word Status` );
		}
		return label.includes( '[Status]' );
	} );
```

Reply:
True. What would you think of something like this?
aec31971e948ff80bfa5e641393f626f1ddba9bc